contrib/uservices/badge/src/schema.rs

```diff
@@ -0,0 +1,12 @@
+table! {
+    run (id) {
+        id -> Int8,
+        run_id -> Int8,
+        num -> Int8,
+        project_key -> Text,
+        workflow_name -> Text,
+        branch -> Nullable<Text>,
+        status -> Text,
```

Comment:
You can use [diesel-derive-enum](https://crates.io/crates/diesel-derive-enum) to store status as enum